mobile/android/base/android_strings.dtd

```diff
@@ -111,15 +111,19 @@
 <!ENTITY pref_category_vendor2 "&vendorShortName; &brandShortName;">
 <!ENTITY pref_category_vendor_summary2 "O programie &brandShortName;, najczęściej zadawane pytania, opinie">
 <!ENTITY pref_category_datareporting "Udostępnianie danych">
+<!ENTITY pref_category_logins "Dane logowania">
 <!ENTITY pref_learn_more "Więcej informacji">
 <!ENTITY pref_category_installed_search_engines "Zainstalowane wyszukiwarki">
 <!ENTITY pref_category_add_search_providers "Dodawanie wyszukiwarek">
 <!ENTITY pref_category_search_restore_defaults "Przywróć wyszukiwarki">
 <!ENTITY pref_search_restore_defaults "Przywróć domyślne">
 <!ENTITY pref_search_restore_defaults_summary "Przywróć domyślne">
-<!ENTITY pref_search_hint "Rada: Możesz dodać każdą witrynę do swojej listy wyszukiwarek, przytrzymując na niej pole wyszukiwania, a następnie stukając ikonę &formatI;.">
+<!ENTITY pref_search_hint2 "Rada: Możesz dodać każdą witrynę do swojej listy wyszukiwarek, przytrzymując na niej pole wyszukiwania, a następnie dotykając ikony &formatI;.">
 <!ENTITY pref_category_advanced "Zaawansowane">
 <!ENTITY pref_category_advanced_summary2 "Przywracanie kart, wtyczki, narzędzia dla programistów">
+<!ENTITY pref_category_notifications "Powiadomienia">
+<!ENTITY pref_content_notifications "Aktualizacje stron">
+<!ENTITY pref_content_notifications_summary2 "Nowości z obsługiwanych stron">
```

Comment:
I don't think it would make good localization for this view:
![screenshot](https://bug1247788.bmoattachments.org/attachment.cgi?id=8737125)

I wouldn't follow en-US so closely here.